flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -492,9 +491,9 @@ under the License.
 						<configuration>
 							<shadeTestJar>false</shadeTestJar>
 							<artifactSet>
-								<includes>
```

Comment:
upon looking at the mvn shade plugin, I think we probably don't even need it anymmore, and can remove this whole section. Can you double check?

Reply:
Yeah I think we can remove it. The shaded jar actually only contains classes from connector-hive